Formula/p/pake.rb

```diff
@@ -0,0 +1,25 @@
+require "language/node"
+
+class Pake < Formula
+  desc "Turn any webpage into a desktop app with Rust"
+  homepage "https://github.com/tw93/pake#readme"
+  url "https://registry.npmjs.org/pake-cli/-/pake-cli-2.3.6.tgz"
+  sha256 "363c1a22cac7266aca44bf27759458da49365ab0df4daad7812070e9c61559e4"
+  license "MIT"
+
+  depends_on "node"
+
+  def install
+    system "npm", "install", *Language::Node.std_npm_install_args(libexec)
+    bin.install_symlink Dir["#{libexec}/bin/*"]
+
+    # Delete native binaries installed by npm, as we dont support `musl` for a `libc` implementation
+    node_modules = libexec/"lib/node_modules/pake-cli/node_modules"
+    (node_modules/"@tauri-apps/cli-linux-x64-musl/cli.linux-x64-musl.node").unlink if OS.linux?
+  end
+
+  test do
+    output = shell_output("#{bin}/pake --version")
+    assert_match "2.3.6", output
```

Comment:
Could you try with a test that covers some of its functionalities? Checks on version or help text almost never fail, even for broken installations. You may check out our [Formula Cookbook](https://docs.brew.sh/Formula-Cookbook#add-a-test-to-the-formula) for some useful advice.

Reply:
Not easily, the test environment doesn't allow pake to set up it's environment on the first run. 

As is stated on their repo: `It might take a while to prepare the environment the first time you launch Pake.

As this formulae is used to create web-apps, I set up a quick test to mock `pake https://brew.sh --name homebrew` but it fails with permissions issues within the test environment like so:

```
npm ERR! code EPERM
npm ERR! syscall mkdir
npm ERR! path /usr/local/Cellar/pake/2.3.6/libexec/lib/node_modules/pake-cli/node_modules/@rollup/rollup-android-arm-eabi
npm ERR! errno -1
npm ERR! Error: EPERM: operation not permitted, mkdir '/usr/local/Cellar/pake/2.3.6/libexec/lib/node_modules/pake-cli/node_modules/@rollup/rollup-android-arm-eabi'
```

While I agree the --version is not a great test, I struggle to come up with another suitable test. Any thoughts are greatly appreciated.

Reply:
That failure means that any user would create files in the install location, which is actually a great test since it indicates a problem with the install.